### ISSUE TITLE
how to get the length of the string in protractor

### ISSUE THREAD

**Issue by peterprotractor** [angular/protractor#2849]
hi ,
i am new to protractor and javascript, i am able to extract the alt text (string) of image by using getAttribute('alt') function, but cannot find the length of that string. how can i do that please advise.

**Comment by szikszail** [comment]
Hi,

E.g:
```javascript
myElement.getAttribute('alt').then(function(altValue){
    return altValue.length;
});
```
http://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.getAttribute

**Comment by juliemr** [comment]
Please direct general support questions like this one to an appropriate support channel, see https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#questions

Thank you!
